### PR TITLE
Chore: Update projects metadata

### PR DESCRIPTION
**Pull request by devendran-m**
To delete obsolete projects and update metadata of some existing projects as outlined by @NicolasZoellner 

Changelog;
```bash
deleted:    ecosystem/air_ballerz/assets/logo.png
        deleted:    ecosystem/air_ballerz/data.md
        modified:   ecosystem/apoc/data.md
        deleted:    ecosystem/artic/data.md
        modified:   ecosystem/ascendex/data.md
        modified:   ecosystem/aws/data.md
        modified:   ecosystem/bitgo/data.md
        deleted:    ecosystem/bithotel_/assets/logo.png
        deleted:    ecosystem/bithotel_/data.md
        modified:   ecosystem/bithumb/data.md
        modified:   ecosystem/bitmart/data.md
        modified:   ecosystem/bitrue/data.md
        deleted:    ecosystem/bixin/assets/logo.png
        deleted:    ecosystem/bixin/data.md
        deleted:    ecosystem/bkex/assets/logo.png
        deleted:    ecosystem/bkex/data.md
        deleted:    ecosystem/blockchain_authenticator/data.md
        deleted:    ecosystem/bw_com/assets/logo.png
        deleted:    ecosystem/bw_com/data.md
        modified:   ecosystem/casper_analytics/data.md
        deleted:    ecosystem/casper_blockchain/assets/logo.png
        deleted:    ecosystem/casper_blockchain/data.md
        modified:   ecosystem/casper_live/data.md
        modified:   ecosystem/casper_qrcode_generator/data.md
        modified:   ecosystem/casper_signer/data.md
        modified:   ecosystem/casper_stats/data.md
        modified:   ecosystem/casper_wallet/data.md
        modified:   ecosystem/casperdash/data.md
        modified:   ecosystem/casperpunks/data.md
        deleted:    ecosystem/caspersign/assets/logo.png
        deleted:    ecosystem/caspersign/data.md
        modified:   ecosystem/casperswap/data.md
        modified:   ecosystem/caspertigers/data.md
        modified:   ecosystem/changenow/data.md
        deleted:    ecosystem/civic/assets/logo.png
        deleted:    ecosystem/civic/data.md
        modified:   ecosystem/coinlist/data.md
        deleted:    ecosystem/coinsuper/data.md
        deleted:    ecosystem/covalent/assets/logo.png
        deleted:    ecosystem/covalent/data.md
        modified:   ecosystem/cspr_market/data.md
        modified:   ecosystem/dario_de_siena/data.md
        deleted:    ecosystem/devxdao/assets/logo.png
        deleted:    ecosystem/devxdao/data.md
        modified:   ecosystem/dotoracle/data.md
        modified:   ecosystem/eggforce/data.md
        modified:   ecosystem/friendly_market/data.md
        modified:   ecosystem/gate_io/data.md
        deleted:    ecosystem/gitcoin/assets/logo.png
        deleted:    ecosystem/gitcoin/data.md
        deleted:    ecosystem/hoo/assets/logo.png
        deleted:    ecosystem/hoo/data.md
        deleted:    ecosystem/hotbit/assets/logo.png
        deleted:    ecosystem/hotbit/data.md
        modified:   ecosystem/huobi/data.md
        modified:   ecosystem/hyperledger_foundation/data.md
        modified:   ecosystem/kucoin/data.md
        deleted:    ecosystem/kunft_marketplace/assets/logo.png
        deleted:    ecosystem/kunft_marketplace/data.md
        modified:   ecosystem/latoken/data.md
        deleted:    ecosystem/metis/assets/logo.png
        deleted:    ecosystem/metis/data.md
        modified:   ecosystem/mexc_exchange/data.md
        modified:   ecosystem/minteandome/data.md
        modified:   ecosystem/nucleus_finance/data.md
        modified:   ecosystem/okx/data.md
        deleted:    ecosystem/parsiq/assets/logo.png
        deleted:    ecosystem/parsiq/data.md
        modified:   ecosystem/robotcache/data.md
        modified:   ecosystem/scansper/data.md
        deleted:    ecosystem/sequoir/assets/logo.png
        deleted:    ecosystem/sequoir/data.md
        modified:   ecosystem/uphold/data.md
        modified:   ecosystem/validation_cloud/data.md
        modified:   ecosystem/waterdrip_capital/data.md
        modified:   ecosystem/wise_art/data.md
        modified:   ecosystem/xt_com/data.md
        deleted:    ecosystem/zb_com/assets/logo.png
        deleted:    ecosystem/zb_com/data.md

```

### CODE REVIEW

**Comment by devendran-m** [comment]
Hi @NicolasZoellner , 
I have cut the PR based on our discussions and the spreadsheet that you shared. Could you review and provide your approvals.
cc: @piotr-dziubecki , @javier-casper 

Hi @alejandro-blockbites ,
I have updated the metadata based on an updated direction received from Nicolas and team. Could you also review it from a technical and segmentation mapping perspective, please?

File: ecosystem/eggforce/data.md
Comment (NicolasZoellner) [inline ecosystem/eggforce/data.md:1]: eggforce is live, please change status

**Review by NicolasZoellner — CHANGES_REQUESTED** [review]
See my comments, thanks

File: ecosystem/casper_wallet/data.md
[the remark this answers is not in the document]
Comment (devendran-m) [reply ecosystem/casper_wallet/data.md:1]: Fixed

File: ecosystem/casperswap/data.md
[the remark this answers is not in the document]
Comment (devendran-m) [reply ecosystem/casperswap/data.md:1]: Incorporated the changes

**Review by NicolasZoellner — COMMENTED** [review]
Thank you very much! Looks good

**Review by NicolasZoellner — APPROVED** [review]

**Review by alejandro-blockbites — APPROVED** [review]
LGTM

**Pull request merged**